sql/core/src/main/scala/org/apache/spark/sql/DataFrame.scala

```diff
@@ -343,6 +343,35 @@ class DataFrame private[sql](
   }
 
   /**
+   * Inner equi-join with another [[DataFrame]] using the column.
+   * {{{
+   *   // Joining df1 and df2 using the column "user_id"
+   *   df1.join(df2, "user_id")
+   * }}}
+   *
```

Comment:
I would also state that similar to SQL USING the join column will only appear once in the output.